helm/assets/configuration/spec.yaml

```
@@ -0,0 +1,10 @@
+name: helm
+files:
+- name: helm.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

```suggestion
    - template: instances/default
       overrides:
         empty_default_hostname.hidden: true
```

Adding the above override in the spec configuration will hide the option. This will need to be re-sync to reflect changes on `conf.yaml.example`.

---

```
< ddev validate config --sync
Validating default configuration files...
Traceback (most recent call last):
  File "/home/ubuntu/dd/integrations-extras/venv/bin/ddev", line 8, in <module>
    sys.exit(ddev())
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/datadog_checks/dev/tooling/commands/validate/config.py", line 52, in config
    spec_path = get_config_spec(check)
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/datadog_checks/dev/tooling/utils.py", line 156, in get_config_spec
    path = load_manifest(check_name).get('assets', {}).get('configuration', {}).get('spec', '')
  File "/home/ubuntu/dd/integrations-extras/venv/lib/python3.8/site-packages/datadog_checks/dev/tooling/utils.py", line 250, in load_manifest
    return json.loads(read_file(manifest_path).strip())
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 22 column 3 (char 552)

```

```
< cat helm/assets/configuration/spec.yaml
name: helm
files:
- name: helm.yaml
  options:
  - template: init_config
    options:
    - template: init_config/default
  - template: instances
    options:
    - template: instances/default
      overrides:
        empty_default_hostname.required: false
```